### PR TITLE
l10n: v2.44 uk localization update

### PR DESCRIPTION
**Pull request by arkid15r**
2.44 l10n draft

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -20651,6 +20765,9 @@ msgstr "Застосування автосхову призвело до кон
 msgid "Autostash exists; creating a new stash entry."
 msgstr "Автосхов існує; створення нового запису схову."
 
+msgid "autostash reference is a symref"
+msgstr "autostash посилання є символьним посиланням"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "посилання автосхову є символьним посиланням"
```

File: po/uk.po [hunk 2]
```diff
@@ -20982,14 +21103,14 @@ msgstr "не вдалося перемістити %s на %s"
 msgid "attempt to reinitialize repository with different hash"
 msgstr "спроба переініціалізувати репозиторій з іншим хеш-алгоритмом"
 
+msgid ""
+"attempt to reinitialize repository with different reference storage format"
+msgstr "спроба переініціалізувати репозиторій з іншим форматом сховища"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "спроба переініціалізувати сховище з іншим форматом зберігання"
```

**Review by kasya — COMMENTED** [review]
👍🏼

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: Great suggestion!

**Pull request closed without merging**